global.d.ts

```diff
@@ -5,3 +5,13 @@ declare module 'vue' {
 }
 
 export type Maybe<T> = T | null;
+
+// source: https://github.com/microsoft/TypeScript/issues/43505
+export type NumericRange<
```

Comment:
https://github.com/sindresorhus/type-fest/blob/main/source/int-range.d.ts I think we can use this type utils library, or something similar. Such a library could be useful in other cases.